bloom-housing/bloom
# feat: doorway infra release - banner

This PR addresses [#(779)](https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/metrotranscom/doorway/779)


## Description

Doorway needs an alert banner for tomorrow's maintenance.

## How Can This Be Tested/Reviewed?

Add `MAINTENANCE_WINDOW=2024-08-07 10:00 -07:00,2024-08-08 8:00 -07:00` to public env file. View public sight and confirm banner is present in all languages.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
